javaslang/src/main/java/javaslang/control/GwtIncompatible.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
Hi @tbroyer, do you have an advise on the best practice how to include the `@GwtIncompatible` annotation into a library that cannot depend on 3rd party libs.

Are we allowed to copy the source-code of the annotation, preserving the copyright file header _without_ including additional LICENSE files?